templates/unix-qa-stage.yml

```diff
@@ -49,7 +49,9 @@ stages:
             - powershell: $(Build.SourcesDirectory)/scripts/ci-self-signed-certificate.ps1
               displayName: "Create self-signed certificate"
             # On Linux/MacOS we need to update the execution permissions for the ScannerCLI
-            - script: chmod -R +x $(Build.SourcesDirectory)/build/sonarscanner-net/sonar-scanner-*/bin
+            - script: |
+                chmod -R +x $(Build.SourcesDirectory)/build/sonarscanner-net/sonar-scanner-*/bin
+                exit 0 # Always return 0 chmod fails when the scanner-cli is not present.
```

Comment:
See here:
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=125550&view=logs&j=5c07e220-b366-5067-695d-b16073551361&t=7b95fc46-99ac-5adb-0485-39859d271b52&l=12

Prevents this failure:
https://dev.azure.com/sonarsource/DotNetTeam%20Project/_build/results?buildId=125544&view=logs&j=5c07e220-b366-5067-695d-b16073551361&t=7b95fc46-99ac-5adb-0485-39859d271b52&l=16